src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -37,16 +37,21 @@ interface BuildLocator {
      * By default only successful builds are returned, call this method to include failed builds as well.
      */
     fun withAnyStatus() : BuildLocator
-
     fun withStatus(status: BuildStatus): BuildLocator
+
     fun withTag(tag: String): BuildLocator
 
-    fun withBranch(branch: String): BuildLocator
+    /**
+     * By default failed to start builds are filtered out, call this method to include them.
+     */
+    fun withAnyFailedToStart(): BuildLocator
```

Comment:
'withAnyFailedToStart' name is very confusing. With what? What do you mean be 'any failed'? I would name the method 'includeFailedToStart', but maybe it would be even better to include FAILED_TO_START into BuildStatus. What do you think?